main_service/queries/course_queries.py

```diff
@@ -1,5 +1,7 @@
 from config import db
 from models.course import Course
+from .constants import courses_per_page
+from math import ceil
 
 def does_course_already_exists(title) :
```

Comment:
je l ne postoji drugi način da vrati samo jedno ako je vec title jedinstven umesto sto vraca listu pa uzimas sa indeksom 0? Ja znam da može sa id ima kod mene u repository u integration service.